.travis.yml

```diff
@@ -35,16 +35,16 @@ language: python
 matrix:
   include:
     - python: 2.6
-      env: NUMPY=numpy==1.6 MOCK=mock
+      env: NUMPY=numpy==1.6
     - python: 2.7
-      env: MOCK=mock PANDAS=pandas
+      env: PANDAS=pandas MOCK=mock
     - python: 3.3
     - python: 3.4
     - python: 3.5
     - python: 2.7
       env: TEST_ARGS=--pep8
     - python: 2.7
-      env: BUILD_DOCS=true MOCK=mock
+      env: BUILD_DOCS=true
```

Comment:
:+1: on switching doc building to 3.5, now that you fixed the numpy related issues that was in the back of my head.